configurationgovernancev1/configuration_governance_v1_integration_test.go

```diff
@@ -178,7 +178,7 @@ var _ = Describe(`ConfigurationGovernanceV1 Integration Tests`, func() {
 
 			excludedScopeModel := &configurationgovernancev1.RuleScope{
 				Note:      core.StringPtr("My account resource group"),
-				ScopeID:   core.StringPtr("4d4daac5f6fd4eb8b9483b97edcb0535"),
+				ScopeID:   core.StringPtr("f8c59a1bc7f04597af0dc6cf00519074"),
```

Comment:
Let's use env or some global var as the same values is used multiple times

Reply:
okay